ci-operator/config/openshift-qe/ocp-qe-perfscale-ci/openshift-qe-ocp-qe-perfscale-ci-main__rosa-4.14-nightly-x86.yaml

```diff
@@ -68,6 +68,7 @@ tests:
     test:
     - chain: openshift-qe-node-density-heavy
     workflow: rosa-aws-sts
+  timeout: 5h0m0s
```

Comment:
for the individual tests like node-density-heavy and node-density-cni I think this timeout is a bit high, but we can keep it as is to keep consistent

Reply:
+1

Reply:
Agreed, keeping this as "constant" across 24nodes/small-scale testing.